src/block/posts/index.php

```diff
@@ -108,14 +108,23 @@ function generate_render_item_from_stackable_posts_block( $post, $attributes, $t
 
 		// Trim the excerpt.
 		if ( ! empty( $excerpt ) ) {
+			$untrimmed_excerpt = $excerpt;
 			$excerpt = explode( ' ', $excerpt );
 			$trim_to_length = (int) $excerpt_length;
-			if ( count( $excerpt ) > $trim_to_length ) {
+
+			// Check if there are CJK characters.
+			if ( function_exists( 'mb_substr' ) && preg_match_all( "/\p{Han}+/u", $untrimmed_excerpt, $matches ) ) {
+				if ( strlen( $untrimmed_excerpt ) > $trim_to_length ) {
+					// Trim according to string length.
+					$excerpt = mb_substr( $untrimmed_excerpt, 3, $trim_to_length ) . '...';
```

Comment:
Why does the index start with `3`, shouldn't it be `0`?
```suggestion
					$excerpt = mb_substr( $untrimmed_excerpt, 0, $trim_to_length ) . '...';
```